### PR TITLE
chore: claude review improvements

### PR DESCRIPTION
**Pull request by grishasobol**

### CODE REVIEW

File: .github/claude-review-prompt.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2. Analyze the full PR diff.
+
+3. Post a summary comment on the PR with:
+   - What the PR does (2-3 sentences)
+   - Overall assessment
+   - Key concerns (if any), as a bulleted list
+
+4. For each specific finding, create a separate inline review comment on the relevant file and line. Format each inline comment as:
+
+   **severity: critical|high|medium|low**
```
[the remark this answers is not in the document]
Comment (grishasobol) [reply .github/claude-review-prompt.md]: outdated

File: .github/claude-review-prompt.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   ```
+
+5. Severity rules:
+   - **critical**: Logic errors, broken state transitions, security issues, consensus bugs
+   - **high**: Missing tests for behavior changes, incorrect API usage, race conditions
+   - **medium**: Missing edge case handling, suboptimal patterns, documentation gaps
+   - **low**: Minor improvements, optional refactors
+
+6. Anti-noise rules (from styleguide):
+   - Do NOT comment on formatting already enforced by rustfmt or forge fmt
```
[the remark this answers is not in the document]
Comment (grishasobol) [reply .github/claude-review-prompt.md]: outdated

File: .github/claude-review-prompt.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   - Do NOT review generated JSON or ABI files directly
+   - Do NOT suggest broad refactors without clear correctness benefit
```
[the remark this answers is not in the document]
Comment (grishasobol) [reply .github/claude-review-prompt.md]: outdated

File: .github/claude-review-prompt.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+   - Do NOT flood with many small comments when one high-signal comment is enough
+   - Do NOT comment on naming, wording, or docs style unless behavior is misleading
```
[the remark this answers is not in the document]
Comment (grishasobol) [reply .github/claude-review-prompt.md]: outdated
